website/docs/cairo-1/06-testing/cheatcodes-reference/start_spoof.md

```diff
@@ -0,0 +1,91 @@
+# `start_spoof`
+
+```cairo
+fn start_spoof(contract_address: felt252, mock: TxInfoMock) nopanic;
+```
+
+Changes `TxInfo` returned by `get_tx_info()` for the targeted contract until the spoof is stopped
+with [stop_spoof](./stop_spoof.md).
+
+- `contract_address` address of the contract for which `get_tx_info()` result will be mocked.
+- `TxInfoMock` collection of fields equivalent to those in `TxInfo` (returned by `get_tx_info()`).
+All fields are optional to allow partial mocking of the `TxInfo`, i.e. for those fields that are `Option::None`, `get_tx_info` will return original values.
```

Comment:
Lets just call it a struct since it is just that. 

Also I'd add a better explanation on how passing `Option::None` works in term of not mocking specific fields or cancelling the mocks.